backend/routes.js

```diff
@@ -130,6 +170,26 @@ export default class Server {
             file: 'oauth.html'
         }));
 
+        server.get(/\/login\/?$/, restify.serveStatic({
+            directory: `${__dirname}/../static`,
+            file: 'login.html'
+        }));
```

Comment:
from a separate comment - can we remove this as just use `/`? it'll be easier if we just have 1 place to direct users